javascript/ql/test/query-tests/Security/CWE-079/tst3.js

```diff
@@ -4,6 +4,7 @@ var data = JSON.parse(decodeURIComponent(window.location.search.substr(1)));
 foo.setAttribute("src", data.src); // NOT OK
 foo.setAttribute("HREF", data.p);  // NOT OK
 foo.setAttribute("width", data.w); // OK
+foo.setAttribute("xlink:href", data.p) // NOT OK
```

Comment:
I assume one could also set this attribute using `setAttributeNS`? If so, could you add a test for that?

Reply:
Yes, one could do that. 

The implementation actually detected that correctly before this change, but I added a test for it.